doc/nrf/releases_and_maturity/migration/migration_guide_2.5.rst

```diff
@@ -27,6 +27,9 @@ The following changes are mandatory to make your application work in the same wa
     UART is now configured using only devicetree.
   * The :kconfig:option:`CONFIG_SLM_CARRIER_APP_DATA_CONTAINER_BUFFER_LEN` Kconfig option has been renamed to :kconfig:option:`CONFIG_SLM_CARRIER_APP_DATA_BUFFER_LEN`.
 
+* Applications using MCUboot must update their version from Kconfig ``CONFIG_MCUBOOT_IMAGE_VERSION`` to using an :ref:`application VERSION file <zephyr:app-version-details>` to set the version.
```

Comment:
```suggestion
* Applications using MCUboot must update to setting their version  using an :ref:`application VERSION file <zephyr:app-version-details>` instead of the previously used ``CONFIG_MCUBOOT_IMAGE_VERSION`` Kconfig option.
```

Reply:
Updated. Any reason as to why this part appears bold? https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12750/nrf/releases_and_maturity/migration/migration_guide_2.5.html

Reply:
One too many spaces in the beginning of the second line, so the build system takes it as a term definition.